src/Nest/Mapping/Types/Specialized/Attachment/AttachmentProperty.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace Nest
 {
+	[Obsolete("Removed in 6.0.")]
```

Comment:
I think we can help users here by pointing them to using the `ingest-attachment` plugin in the comment.

Reply:
:+1: